core/version/gear.ex

```diff
@@ -29,15 +29,17 @@ defmodule AntikytheraCore.Version.Gear do
       if :code.get_mode() != :interactive do
         # Load module manually
         Enum.each(Application.spec(gear_name, :modules), fn(mod) ->
-          # Croma defines modules (e.g. `Elixir.Croma.TypeGen.Nilable.Antikythera.Email`) automatically in the gear.
-          # As a result, some modules which have the same name are defined in two or more gears.
-          # We have to avoid loading these modules twice.
-          skip_to_load? = can_skip_to_load?(mod) && :code.is_loaded(mod)
-          if not skip_to_load? do
-            case :code.load_file(mod) do
-              {:module, _}      -> :ok
-              {:error , reason} -> raise "Failed to load '#{mod}': #{reason}"
-            end
+          case :code.load_file(mod) do
+            {:module, _}      -> :ok
+            {:error , reason} ->
+              # Croma defines modules (e.g. `Elixir.Croma.TypeGen.Nilable.Antikythera.Email`) automatically in the gear.
+              # As a result, some modules which have the same name are defined in two or more gears, and loading these modules causes an error.
```

Comment:
(optional)

```suggestion
              # As a result, some modules with the same name are defined in two or more gears, and loading these modules causes an error.
```

I think this is easier to read.